examples/i-inverse-kinematics.cpp

```diff
@@ -1 +1,33 @@
-// write cpp example here
+#include "pinocchio/multibody/model.hpp"
+#include "pinocchio/parsers/urdf.hpp"
+#include "pinocchio/algorithm/jacobian.hpp"
+#include "pinocchio/algorithm/joint-configuration.hpp"
+
+int main(int argc, char ** argv)
+{
+  std::string filename = (argc<=1) ? "ur5.urdf" : argv[1];
+  se3::Model model;
+  se3::urdf::buildModel(filename,model); 
+  se3::Data data(model);
+
+  const int    JOINT_ID = 6;
```

Comment:
For easy use, I would look the joint_id by its name in the kinematic tree. 
This is the standard way of doing it.

Reply:
IK example is not mature. I just kept it for future edition.

Reply:
I think that for a 6D fixed-base manipulator saying "joint 6" is much easier and clearer than "wrist_3_joint"